pkg/config/ingress.go

```diff
@@ -14,6 +14,17 @@ type IngressConfig struct {
 }
 
 type RouteAdmissionPolicy struct {
+	// namespaceOwnership describes how host name claims across namespaces should
```

Comment:
```suggestion
	// NamespaceOwnership describes how host name claims across namespaces should
```

Reply:
There is this discrepancy, but I saw it in other parameters. Let me rephrase it to avoid mentioning it (its redundant anyway), as this is also quite common in other parameters.